src/main/java/uk/ac/ebi/embl/converter/utils/ConversionEntry.java

```diff
@@ -10,30 +10,59 @@
  */
 package uk.ac.ebi.embl.converter.utils;
 
+import io.vavr.Tuple2;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 public class ConversionEntry {
     String sOID;
     String sOTerm;
     String definition;
     String feature;
-    String qualifier1;
-    String qualifier2;
+    Map<String, String> qualifiers;
 
     ConversionEntry(String[] tokens) {
         this.sOID = tokens[0];
         this.sOTerm = tokens[1];
         this.definition = tokens[2];
         this.feature = tokens[3];
-        for (String token : tokens)
-            if (tokens.length > 4) this.qualifier1 = tokens[4].equalsIgnoreCase("null") ? null : tokens[4];
-        if (tokens.length > 5) this.qualifier2 = tokens[5].equalsIgnoreCase("null") ? null : tokens[5];
+        this.qualifiers = new HashMap<>();
+        List<String> remainingTokens = Arrays.stream(tokens).skip(4).toList();
+        for (String token : remainingTokens) {
+            Tuple2<String, String> parsed = parseQualifier(token);
```

Comment:
We can use Map instead of Tuple, also added comment to update the method `parseQualifier(token);`
```
List<String> qualifiersTokens = Arrays.stream(tokens).skip(4).toList();
        for (String token : qualifiersTokens) {
            Map<String, String> parsedQualifiers = parseQualifier(token);
            if (parsedQualifiers != null) {
                qualifiers.putAll(parsedQualifiers);
            }
        }
```